src/main/resources/assets/computercraft/lua/rom/apis/colors

```diff
@@ -25,13 +25,23 @@ function combine( ... )
 end
 
 function subtract( colors, ... )
-	local r = colors
-	for n,c in ipairs( { ... } ) do
-		r = bit32.band(r, bit32.bnot(c))
-	end
-	return r
+    local r = colors
+    for n,c in ipairs( { ... } ) do
+        r = bit32.band(r, bit32.bnot(c))
+    end
+    return r
 end
 
 function test( colors, color )
     return ((bit32.band(colors, color)) == color)
 end
+
+function rgb8( r, g, b )
```

Comment:
It's intended to be plugged into `term.setPaletteColour` so you can specify each channel as an 8-bit number:
```Lua
term.setPaletteColour(colours.white, colours.rgb8(255, 0, 255))
```
I added it as a compromise for the  0-1 range vs. 0-255 range debate.

Just to clarify, the current possibilities are:
```Lua
term.setPaletteColour(colours.white, 0.0, 1.0, 1.0)

-- I can't make an overload for this, hence the addition of colours.rgb8
term.setPaletteColour(colours.white, colours.rgb8(0, 255, 255)) 

term.setPaletteColour(colours.white, 0x00FFFF)
```